packages/stream-server-uws/stream_server_uws.js

```diff
@@ -0,0 +1,90 @@
+// noinspection NpmUsedModulesInstalled
+const { RoutePolicy } = require('meteor/routepolicy');
+// noinspection NpmUsedModulesInstalled
+const { WebApp } = require('meteor/webapp');
+
+// Try load of `uws` Node.js package
+/** @type {WebSocket} */
+let WebSocket;
+try {
+  // noinspection NpmUsedModulesInstalled
+  WebSocket = require('sc-uws');
+} catch (e) {
+  console.warn('Run:\nmeteor npm install sc-uws')
+}
+
+// noinspection JSUnresolvedVariable
+const pathPrefix = __meteor_runtime_config__.ROOT_URL_PATH_PREFIX;
+const PATH = '/websocket';
+const UWS_SERVER_OPTIONS = {
+  path: pathPrefix + PATH,
+  server: WebApp.httpServer,
+};
+
+/**
+ * StreamServer with uws
+ * @class
+ * @type {StreamServerUWS}
+ */
+StreamServerUWS = class StreamServerUWS {
```

Comment:
If you use the `ecmascript` package, this can just be
```js
export class StreamServerUWS {
```

Reply:
I wrote more about this in the comment:
https://github.com/meteor/meteor/pull/10120#discussion_r207611025